autolens/plot/plotters/fit_imaging_plotters.py

```diff
@@ -0,0 +1,292 @@
+from autoarray.plot.plotters import abstract_plotters
+from autoarray.plot.plotters import inversion_plotters
+from autoarray.plot.plotters import fit_imaging_plotters
+from autolens.plot.plotters import ray_tracing_plotters
+from autogalaxy.plot.mat_wrap import lensing_mat_plot, lensing_include, lensing_visuals
+from autolens.fit import fit as f
+
+import numpy as np
+
+
+class FitImagingPlotter(fit_imaging_plotters.FitImagingPlotter):
+    def __init__(
+        self,
+        fit: f.FitImaging,
+        mat_plot_2d: lensing_mat_plot.MatPlot2D = lensing_mat_plot.MatPlot2D(),

```

Comment:
Could be worth doing something like ag.MatPlot2D for this types as this function signature is looking pretty noisy